src/components/Alphabet/AlphabetPlaceholder.js

```diff
@@ -8,115 +8,115 @@ function AlphabetPlaceholder() {
         <SectionTitle.Presentation placeholder />
       </div>
       <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
-        <div className="grid grid-cols-7 gap-8 md:divide-x-2 divide-gray-100">
+        <div className="grid grid-cols-7 gap-8 md:divide-x-2 divide-charcoal-50">
           <div className="col-span-7 md:col-span-4">
             <div className="grid grid-cols-7 md:grid-cols-5 lg:grid-cols-7">
-              <span className="bg-gray-100 text-gray-100 col-span-1 font-medium inline-flex justify-center m-1 p-5 rounded text-2xl">
+              <span className="bg-charcoal-50 text-charcoal-50 col-span-1 font-medium inline-flex justify-center m-1 p-5 rounded text-2xl">
```

Comment:
minor: we can make this style reusable similar to btn-icon for this file

Reply:
Reduction in duplication is definitely warranted here, but I think a local variable would be the way to go. Global classes, like `btn-icon`, are more for class combinations that are reused across many files and for styling that we would potentially want to change at a global level.